tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -0,0 +1,72 @@
+from marionette import Marionette
+from gaiatest.apps.base import Base
+from marionette_driver import By, Wait, expected
+
+class BugzillaLite(Base):
+
+    name = 'Bugzilla Lite'
+    _given_username = (By.CSS_SELECTOR, "#login input[type='email']")
+    _given_password = (By.CSS_SELECTOR, "#login input[type='password']")
+    _button_login = (By.CSS_SELECTOR, "#login input[type='submit']")
+    _profile_icon = (By.CSS_SELECTOR, ".profile")
+    _button_logout = (By.CSS_SELECTOR, ".btn")
+    _dashboard_navigator_locator = (By.ID, 'dashboardNav')
+    _dashboard_login_locator = (By.ID, 'login')
+    _popup_intro = (By.ID, 'intro')
+    _button_popup_intro = (By.ID, 'intro-submit')
+    _assigned_icon = (By.CSS_SELECTOR, '#dashboardNav .assigned')
+    _flagged_icon = (By.CSS_SELECTOR, '#dashboardNav .flagged')
+    _filed_icon = (By.CSS_SELECTOR, '#dashboardNav .filed')
+    _search_bar = (By.ID, 'searchLink')
+    _button_cancel_search = (By.LINK_TEXT, 'Cancel')    
```

Comment:
I haven't looked to the DOM but if we don't have any other way to select this button, that'd be a bug to file to BzLite devs.